modules/openebs/main.tf

```diff
@@ -0,0 +1,26 @@
+# Create OpenEBS namespace
+resource "kubernetes_namespace" "openebs" {
+  count = var.install_openebs ? 1 : 0
+
+  metadata {
+    name = var.openebs_namespace
+  }
+}
+
+# Install OpenEBS Helm chart for lgalloc support
+resource "helm_release" "openebs" {
+  count = var.install_openebs ? 1 : 0
```

Comment:
If all objects in the module are gated by this, why not put this at the module instantiation? There is no reason to even have this variable.